tests/Mock/MockWcOrder.php

```diff
@@ -35,4 +35,22 @@ public function add_order_note(string $note, int $is_customer_note = 0, bool $ad
 
         return count($notes) - 1;
     }
+
+    /**
+     * @return string
+     */
+    public function get_status(): string
```

Comment:
deze functie is niet nodig, als je per se defaults wil wijzigen kun je dit doen:

```
protected $attributes = [
  'status' => '',
];
```

maar check eerst maar of dat echt nodig is